tests/queries/0_stateless/03417_http_exception_tag.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+
+CLICKHOUSE_DATABASE="default"
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+echo "Test 1: Check that X-ClickHouse-Exception-Tag header is present"
+${CLICKHOUSE_CURL} -sS -D - "${CLICKHOUSE_URL}" --data-binary "SELECT 1" | grep "X-ClickHouse-Exception-Tag" | head -1 | cut -d':' -f1 | sed 's/$/:/g'
+
+echo "Test 2: Verify exception tag is alphanumeric"
+TAG=$(${CLICKHOUSE_CURL} -sS -D - "${CLICKHOUSE_URL}" --data-binary "SELECT 1" | grep "X-ClickHouse-Exception-Tag" | cut -d':' -f2 | tr -d ' \r\n')
+if [[ "$TAG" =~ ^[A-Za-z0-9]+$ ]] && [[ ${#TAG} -eq 8 ]]; then
```

Comment:
I've noticed that getRandomASCIIString (unlike FunctionRandomPrintableASCII) only uses `[a-z]`, which means 26^8 = 37.6 bits of entropy, which is probably good enough, but it's much less than UUIDv4 has, for example

Reply:
Why should we care about collisions between different requests?

The main purposeof having a random string is to be able to distinguish the exception marker from the data in the response. It's already quite unlikely to find the same random string in the data, but why not make it longer?